### ISSUE TITLE
在微信小程序中如何使用呢？

### ISSUE THREAD

**Issue by xiaobo1215** [mqttjs/MQTT.js#733]
想请问一下，这个在微信小程序中如何调用呢？

**Comment by mcollina** [comment]
I’m sorry, can you rephrase in English?

**Comment by 4rzael** [comment]
Google translates it to : "How to use in WeChat applet?"

I never looked at wechat applets, but apparently, you can use any js library.

So, just install it, and, apparently, you will want to make the connection in the `onLaunch` function, and then use the `publish` and `subscribe` functions however you want. It does not seem to be very different from any javascript code.

Can you explain your problem more specifically, please ?

**Comment by xiaobo1215** [comment]
Thank you for your answer  My question is how to use emqtt in WeChat

**Comment by xiaobo1215** [comment]
is WeChat mini apps

**Comment by xiaobo1215** [comment]
So just to ask, how do we call this in a WeChat mini apps?

**Comment by MY201314MY** [comment]
And have you solved the problem?I also want to use the library?
